projects/plugins/jetpack/extensions/blocks/button/button.php

```diff
@@ -39,9 +39,7 @@ function register_block() {
 function render_block( $attributes, $content ) {
 	$save_in_post_content = get_attribute( $attributes, 'saveInPostContent' );
 
-	if ( Blocks::is_amp_request() ) {
-		Jetpack_Gutenberg::load_styles_as_required( FEATURE_NAME );
-	}
+	Jetpack_Gutenberg::load_styles_as_required( FEATURE_NAME );
```

Comment:
Previously we were only loading the `view.scss` styles for amp requests. There are no other pre-existing stylesheets for the button on the frontend where I could put the alignment fixes; I'm still a little shaky on best practices here so I'm definitely looking for feedback if there's a better place to do this.